content/en/faq/metrics-and-logs/metric-expiry.md

```diff
@@ -0,0 +1,15 @@
+---
+title: How to work around missing metric expiry with in-proxy telemetry?
```

Comment:
```suggestion
title: How can I manage metrics for short-lived jobs?
```

Reply:
Updated, although it is not only for short lived jobs. It can be short lived deployment, or traffic goes out of the mesh and has random host header. So I make it `How can I manage short-lived metrics?`.